cartoframes/viz/layer.py

```diff
@@ -104,7 +112,9 @@ def __init__(self,
 
 def _set_source(source, context):
     """Set a Source class from the input"""
-    if isinstance(source, (str, list, dict, Dataset)):
+    if isinstance(source, (str, list, dict, Dataset)) or \
+       isinstance(source, pandas.DataFrame) or \
```

Comment:
Why not include `pandas.DataFrame` in the tuple above with the other types?

Reply:
True that